### ISSUE TITLE
TypeError: BaseModel.validate() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by q2wxec** [truera/trulens#742]
when use langchain as a provider,error occour
codes
```
from trulens_eval.feedback.provider.langchain import Langchain
provider = Langchain(chain=llm)
```
stack
```
Traceback (most recent call last):
  File "\codes\ai\langchain\lesson3\trulens.py", line 42, in <module>
    provider = Langchain(chain=llm)
  File "\AppData\Local\Programs\Python\Python310\lib\site-packages\trulens_eval\feedback\provider\langchain.py", line 43, in __init__
    self_kwargs["endpoint"] = LangchainEndpoint(
  File "\AppData\Local\Programs\Python\Python310\lib\site-packages\trulens_eval\feedback\provider\endpoint\langchain.py", line 64, in __init__
    super().__init__(*args, **kwargs)
  File "\AppData\Local\Programs\Python\Python310\lib\site-packages\trulens_eval\feedback\provider\endpoint\base.py", line 173, in __init__
    super(SerialModel, self).__init__(*args, **kwargs)
  File "\AppData\Local\Programs\Python\Python310\lib\site-packages\trulens_eval\utils\pyschema.py", line 610, in __init__
    super().__init__(*args, **kwargs)
  File "\AppData\Local\Programs\Python\Python310\lib\site-packages\pydantic\main.py", line 164, in __init__
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
TypeError: BaseModel.validate() takes 2 positional arguments but 3 were given
```

**Comment by joshreini1** [comment]
Looking into this

**Comment by joshreini1** [comment]
@q2wxec - this was fixed by https://github.com/truera/trulens/pull/759 and merged to main. Will be released in 0.20.1 shortly
